docs/char-info-str.md

```diff
@@ -30,7 +30,7 @@ api_type:
 - HeaderDef
 ---
 
-# `CHAR\_INFO structure`
+# CHAR\_INFO structure
```

Comment:
isn't it technically more correct to put the backticks around the struct name only? Or is that wrong?

Reply:
It is technically more correct but it doesn't render correctly after markdown processing and puts a goofy looking code block on the header line. The other structures don't use the code block and it looks better this way.